src/main/scala/it/pagopa/interop/backendforfrontend/service/types/PurposeProcessServiceTypes.scala

```diff
@@ -134,14 +135,7 @@ object PurposeProcessServiceTypes {
       isFreeOfCharge = p.isFreeOfCharge,
       freeOfChargeReason = p.freeOfChargeReason,
       dailyCallsPerConsumer = currentDescriptor.dailyCallsPerConsumer,
-      dailyCallsTotal = currentDescriptor.dailyCallsTotal,
-      riskAnalysisId = if (eService.mode == CatalogProcess.EServiceMode.RECEIVE) {
```

Comment:
I may be missing a requirement here, but is there a reason for this intersection?